### ISSUE TITLE
Crash Landing (Snow Map) - Project Discussion

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#6789]
Ship's crashlanded on a snow planet with limited resources. Things that need to be coded for it, preliminary discussion
- cut down trees to get wood - @unid15 
- proper snow tiles - @DrCelt
- coldness of snow - @DrCelt
- hypothermia - @DrCelt #7158
- RANDOM BUILDING POOL - @unid15 #8683 
- z-level transition in snow - @DrCelt #7223 
- cannibalism? 

sprites - @Inorien and @Skullyton have agreed to make some. we need more spriters.
- yetis? -
- ship debris. 
- fireplaces / camp fires - @NigglyWiggly & eliminator
- snow coloured minerals - @Skullyton 
- snow mobs 
- snow flora - @Skullyton 
- igloos - 
- expanding @Nienhaus 's paths - @Skullyton 
- snow animations like in snowfx.dm but MORE OF THEM.

map
- MAIN Z-LEVEL crashed ship, maybe the size of meta's escape shuttle with materials sufficient to build a station (maintain minimalism!), - an abandoned cargo outpost with an AI core & mineshaft as well as flora and minerals.
- UNDERGROUND MINE Z-LEVEL - cold, high risk high reward minerals
- TCOMMS-LIKE DUNGEON Z-LEVEL
-  Mappers, you don't even have to map an entire station. Just map a cosy wee cabin with a wee bit of loot maybe a wee bit of lore too, and we'll copy it in to our "random pool"

things to port:
- those really cool /tg/ winter clothes

**Comment by The-Whargoul** [comment]
Dabbling with a map now. unsure what a desire size would be, could just roll with the default. 255x255

**Comment by Probe1** [comment]
sounds like you want to make an entirely new byond game baka desu

**Comment by ghost** [comment]
I have some snow things if you guys want them.
https://www.dropbox.com/s/mlqtuupyq1pc1lt/Snow.dmi?dl=0
https://www.dropbox.com/s/lq7686jotc9pyc7/Snowoverly.dmi?dl=0
https://www.dropbox.com/s/bctb22krbb6ukpn/Snowmobile.dmi?dl=0 (Not gonna lie, not a giant fan of this)
https://www.dropbox.com/s/cbsufnxg877w2on/VertiIceOverlay.dmi?dl=0 (Iced door)
https://www.dropbox.com/s/b98aed2yzcshzbw/HorizIceOverlay.dmi?dl=0 (Iced door)
https://www.dropbox.com/s/f4cu2r1nwdier0u/Galactic%2BMarine.dmi?dl=0 (This was supposed to be some now syndi guy)
https://www.dropbox.com/s/h7luzwf33plgdro/J.dmi?dl=0 (This is some ice stuff)

**Comment by The-Whargoul** [comment]
Neat, Thankyou for the snow stuff

**Comment by ghost** [comment]
I really want snow things to happen. You have no idea.

**Comment by ghost** [comment]
@probe1 
A new map is meant to change the meta. The game modes would remain exactly the same.

**Comment by ihadtoregisterforthis** [comment]
Things to take into account
- Should it be a blank map or have terrain difficulties?
- How will the z-level border be? It can't stay as invincible walls
- What would happen with cargo? At least mining would be easy, get some underground mine
- Turbofreezing pipes would become a lot harder
- Trash disposal/body disposal would be an issue
- Explosions uncover space tiles under the floor, ruining my inmersion
- A bunch of gamemodes would need work, such as vampires, nuke ops, "stealing X" objectives
- Emergency shuttle? On a planet!? AND WE WERE STRANDED FOR NO REASON ALL THIS TIME!?

If the ladders from that away mission can be implemented on this it would be great

**Comment by ghost** [comment]
![iv1ny1c](https://cloud.githubusercontent.com/assets/285438/11461916/be136632-96d1-11e5-92a3-451957f435f5.png)
https://www.dropbox.com/s/noeub6a76quk3gp/Elevator.dmi?dl=0
This could also be used for an underground mining elevator if you want.

**Comment by ghost** [comment]
@ihadtoregisterforthis
The emergency shuttle has crashed, duh!
Also space tiles when destroyed can be changed. 
The rest is up for debate

**Comment by ghost** [comment]
@nienhaus
I can't thank you enough take my gratitude in the form of a heart please thanks ❤

**Comment by The-Whargoul** [comment]
this is supposed to be an event right? not for regular mode rotation?

**Comment by ghost** [comment]
♥
I also have wood stuff if you want to make things build-able out of wood.
https://www.dropbox.com/s/dcdy9n42ce8okso/Wood.dmi?dl=0
https://www.dropbox.com/s/wz0ih0z6ocxa4y0/Wood%202.0.dmi?dl=0

**Comment by ghost** [comment]
@The-Whargoul there's no specific reason it couldn't be in the rotation as far as I can see.

**Comment by ghost** [comment]
I also have some lumberjack stuff that's a year old. It's not great however, the axe and saw look good imo.
https://www.dropbox.com/s/plven5wr3f3k2er/Lumberjack.dmi?dl=0

**Comment by ghost** [comment]
@nienhaus I don't know what to say man, this is above and beyond what I was expecting in the first few minutes of making this issue. I'll try and make a PR to put them in the game.

**Comment by The-Whargoul** [comment]
@DrCelt 
Just would need to know if it's needed to be a 'we build it' sandbox map with resources and places to explore/loot 
or if it's supposed to be an already functioning station on a planet.

**Comment by ghost** [comment]
You're welcome, it helps I get email updates on a lot of servers. As soon as I saw the "SNOW" I looked over this.

**Comment by ghost** [comment]
@The-Whargoul the intention, at least in my eyes, was that it's functional enough to be built into a working station without admin intervention, but is all contained within the "crashed shuttle". In which case, it doesn't matter if it's in the rotation or not.

**Comment by ghost** [comment]
@nienhaus good man yourself. If you find/make any other cool stuff then please do contribute, the artic station dream will be real.

**Comment by ghost** [comment]
If you want to take a look at my sprite here.
https://www.dropbox.com/sh/n3w3d7sxhmz91bs/AAC-xK_rAYFgdirzBEe6Vmz1a?dl=0

**Comment by ghost** [comment]
> body disposal becomes a problem on a planet map
> This is beautiful. Add burying people and things. Any ideas on if blood should be impossible to wash out?

**Comment by ghost** [comment]
Oh, and should there be a krrnk sound every step you take in the snow?

**Comment by FudgePucker** [comment]
There needs to be fire places for maximum comfyness and being able to keep an area warm.

Oh good god I have no idea how you are planning on doing this, you might as well disable atmospherics and just have a jury rigged system in place for heat and shit. (You don't need atmospheric simulation on a planet. They got really lucky and it has the right amount of O2 and N2 and no nasty shit like cyanide in the air.)

The fire places should be plasma based or wood based.

**Comment by Kurfursten** [comment]
Live testing from today suggests to me that things would run smoother if the crashed crew starts with a load of stacked up flatpacks with essentials for each job. It sucks being a botanist and literally never getting a hydroponics tray.

**Comment by Zth--** [comment]
we should add some rotting system for the deads. Like dwarf fortress.
I'm really interested in this, I'm not a coder but I can program some stuff. Let me know if you make a team or something.
Also I love the idea of underground mining

**Comment by FudgePucker** [comment]
@Kurfursten 
Haven't used flatpacks in a while recently (or ever)
Didn't they get updated to have stack-a-bility?

It would be nice to just have stacks of flatpacks for every department, botanist getting a stack of packs containing two trays, the two venders, a seed extractor, (xenobotany if there still is room in the stack of packs of stacks (boy that sounds rally fun to say out loud but looks stupid in text)), cargo could have a stack of packs of the cargo pad (Its like the cargo shuttle only a pad instead of a shuttle) cargo computer and mining redeemer.

Sorry if this comment is a mess, the oxi kicked in and man do I feel like passing out now.

**Comment by The-Whargoul** [comment]
@FudgePucker 

Me and Celt discussed a possible outpost long abandoned. Would be not too far off from the Shuttle crash, and would hold the ore machines and cargo consoles, which could be powered whenever the crew constructs their engine. 
said outpost would not be significant in size, else crew may not be as motivated to create shelter.

**Comment by Iamgoofball** [comment]
if you guys have proper material datums you could do some fun DF style stuff on the planet with ores and wood

**Comment by Kurfursten** [comment]
@FudgePucker Yes

**Comment by FudgePucker** [comment]
@Iamgoofball 
Yea remember how well it went last time we tried impementing 'goofball' '''''''proper'''''''' material datums

:^^^)))))))))))))))))))))))))))))))))))))

**Comment by 9600bauds** [comment]
One problem with this is that wood is practically cosmetic in the game, so chopping tree would be worthless unless you really, really need some tables I guess.

**Comment by Kurfursten** [comment]
Save metal vs using it for floor tiles I suppose, but metal is cheap, esp. with the infinite produced by MoMMIs and engborgs

**Comment by Zth--** [comment]
It would be used for campfires

**Comment by Iamgoofball** [comment]
@FudgePucker i didn't port it here? You're thinking of something else, get your memes right faggot. Plus it never got finished because of the maintainers at tg heavily implied none of them would merge it even if it got finished.

**Comment by ghost** [comment]
Let's not forget to turn off events like radiation, meteors, vines, carp, actually pretty much every event.

**Comment by Inorien** [comment]
As far as the Engineering side of things goes, as we faced in issues yesterday, APC's really won't work without the blueprint system, which I understand is a pain to get working for new rooms, which is all the outpost-base-thing would be. Either this would need a rework or simply be abandoned, and push all power out of SMES. This would work easier by implementing a Taxi-like system of portable SMES to bring power to all regions of the outpost, rather than having a really elaborate cabling system out of Engineering that any clown can chew on and kill the whole outpost. 
For Atmos, anything other than a breathable atmosphere at standard pressure would kill the round before it even begins if no Atmos techs are around. This being said, skipping atmos completely would suck - how about having an O2 miner to fill oxygen tanks and stuff and have more 'inhospitable' regions of the planet - eg. gas swamps which vent plasma, giving a natural source of plasma, or some mine full of some other non-breathable gas. 
Trees were cool on the map. Hatchet + Tree = deconstruct = wood floors?

Also
![front](https://cloud.githubusercontent.com/assets/16082007/11471350/b1492640-975c-11e5-8645-5b6010452da8.png)

?

**Comment by Wizardcrying** [comment]
Is there going to be lighting throughout the entire zlevel? If so, they shouldn't be randomly glowing lights everywhere. We'd need a constructable light source.

**Comment by Inorien** [comment]
^

day/night cycle?

**Comment by Inorien** [comment]
Thoughts on Yeti pls
![back](https://cloud.githubusercontent.com/assets/16082007/11473791/76a5acba-976c-11e5-8585-9233f5d173d7.png)
![front](https://cloud.githubusercontent.com/assets/16082007/11473794/76cd6804-976c-11e5-90b1-1413a775179a.png)
![left](https://cloud.githubusercontent.com/assets/16082007/11473793/76c9d914-976c-11e5-88b8-05b3720b1f95.png)
![right](https://cloud.githubusercontent.com/assets/16082007/11473792/76c8e5ae-976c-11e5-95d8-7476bf84fc5d.png)

**Comment by ihadtoregisterforthis** [comment]
![skifree-360x300](https://cloud.githubusercontent.com/assets/13305885/11476900/f44eb8b8-9763-11e5-99db-e482b057d0ec.jpg)

**Comment by ghost** [comment]
I like it @Inorien , only thing is the face might be a little to dark as I can't tell if there are any features on it.

**Comment by Inorien** [comment]
Will be brightening the skin up and adding blue/grey tints to give more hint of coldness

**Comment by Inorien** [comment]
![front2](https://cloud.githubusercontent.com/assets/16082007/11500638/425ad6c8-9826-11e5-9a14-00999b5075b4.png)
![snowbush1](https://cloud.githubusercontent.com/assets/16082007/11500642/4546ca72-9826-11e5-8dca-3c63f2d4172c.png)
blued-up the yeti and recoloured skin + eyes, also have a snowy bush (looks shit on pure white background)

or would grey skin on the yeti be better

**Comment by ghost** [comment]
Need a blizzard animated sprite with various levels of blizzard from impossible to see past to limited vision. This is for the purpose of z-level transfer.

**Comment by ihadtoregisterforthis** [comment]
Maybe you could whip up one of those fancy scripts like the one for snow in space

**Comment by ghost** [comment]
Idea. Randomly generate hidden buildings from a select few. Opinion? We could have a z-level of possible buildings and steal holodeck code.

**Comment by DrLeprikon** [comment]
I made a fur tree, I'll add in some snow in a while!
![image](https://cloud.githubusercontent.com/assets/10002997/11537851/8a3943cc-9938-11e5-8135-254ba0394447.png)

**Comment by ghost** [comment]
There's already a tree in the game, Boris - pine trees specifically!

**Comment by DrLeprikon** [comment]
Those are old trees! They are associated with evil and you said you wanted to change the meta!
![image](https://cloud.githubusercontent.com/assets/10002997/11537956/05dd0978-9939-11e5-8e63-b9daa46aaf41.png)

**Comment by DrLeprikon** [comment]
This is an item, I call her "Fur Tree Branch"
![image](https://cloud.githubusercontent.com/assets/10002997/11538064/8f357c96-9939-11e5-82d2-f3d87814c0f7.png)

**Comment by FudgePucker** [comment]
11/10 best sprite artist

**Comment by DrLeprikon** [comment]
Thanks, I am a certified drawed in IRL, so this was an easy gig for me

**Comment by ghost** [comment]
They need better shading,  Boris. Also perhaps you should use a better source to base your image on as this is what fir trees look like! https://upload.wikimedia.org/wikipedia/commons/6/6d/Snow_on_fir_trees.jpg

**Comment by ghost** [comment]
@Zth-- Any help is of course appreciated.

**Comment by ghost** [comment]
What needs doing?

**Comment by Inorien** [comment]
SPRITES, EVERYONE SPRITES

FLORA
DEBRIS
SNOWY SHIT
ANIMATED BLIZZARD WITH DIFFERENT INTENSITIES

**Comment by ghost** [comment]
After holding votes in both deadchat and ingame, I have came across the following conclusions, take them as you will
- Most importantly, people are fairly hyped for the idea of a snow station.
- Nienhaus' Snow sprite is considered better than the "old" snow sprite.
- Nienhaus' Wooden door sprite is considered to be better than the "old". but their wooden walls are not.
- People do like the idea of a crunching sound playing as they walk on the snow - with most arguing for around a 25-50% chance.
- People are divided by the idea of a chance of "getting stuck in the snow"
- People do very much like the idea of choosing buildings from a pool to randomly generate on the map.
- People do very much like the idea of an underground mine, some suggest both, with the underground being far more dangerous - hellroid springs to mind
- People are in favour of a snow map "dungeon/castle" like tcomms - but they also warn that this is an extra and should not be focused on right now, and should be very hard to win, and have a great reward.
- People like the idea of both a blizzard random event as well as it passively snowing.

What does this mean?: 
I'll immediately get to work on porting Nienhaus' snow sprite and wooden door, and add a crunching sound for snow. In the same request, I'll make an unsimulated, around -20c snow tile.
Mappers, you don't even have to map an entire station. Just map a cosy wee cabin with a wee bit of loot maybe a wee bit of lore too.
Three z-levels are defined - The mine, the main z-level, the castle/dungeon.

**Comment by Inorien** [comment]
Sounds good. I'll continue throwing things together when I have time, but I do have exams in 2 weeks and those take priority for now. I'll improve the yeti, do more various flora (but we don't want too much - not more than 3-4 differently appearing plants, else it'll be kinda weird) and snowy versions of existing turfs that will be used outdoors, eg floor tiles. Is it feasible to do something like - freshly laid metal floor tiles, after some time outdoors/snow 'event' they get partially snowy, then eventually completely covered in snow? Walking over them would remove one 'tier' of snow or reset the timer on it progressing to the next 'tier' of snow. End result, if tiles are outdoors for too long with no traffic, they become hidden by snow. Idea is to encourage outdoor traffic or regular snow shoveling to avoid paths between buildings being lost (get lost in woods or at an out-of-the-way location, suddenly no clear path back, encounter yeti camp). 

Hot lights - more outposty version of space heater, also provides some illumination, melts snow around it.

Metal walls for buildings - low temperature outside, slightly higher temperature inside. No idea how feasible these would be in BYOND, but it'd be a cool dynamic.

Let's just move away from BYOND and make a proper game engine / port SS13 to a proper game engine that isn't made of cancer

**Comment by ghost** [comment]
@Inorien Unfortunately we can't remake the wheel. Yes, I was considering a blizzard event or some kind of passive snowing.

**Comment by Inorien** [comment]
#YesWeCan 
#MakeSS13GreatAgain

**Comment by Iamgoofball** [comment]
ayy, important question:

did you faglords port the Materials datums from bay or make your own yet

**Comment by ghost** [comment]
@Inorien So by flora, do you mean trees, bushes, tiny trees, grass?

There an image of the ground snow sprite I can use for a background so I can see how it'll be?

**Comment by ghost** [comment]
@Skullyton We've recently voted to change the snow sprite. These are the sprites we intend to change it to, thanks to the lovely @Nienhaus. - https://www.dropbox.com/s/mlqtuupyq1pc1lt/Snow.dmi?dl=0

**Comment by Inorien** [comment]
Exactly Skully, go for bushes, happy little trees, rocks, anything to add some variety to the landscape. But like I said, we don't want too many. If there are tens of different kinds of floor decorations around the outpost, it would not look natural at all

**Comment by Zth--** [comment]
I think 255 x 255 is small for the main map. Can we have a larger one?

**Comment by ghost** [comment]
@Zth-- The idea is the main map is fairly sparse. It'd seem even sparser if it's big, but that's not necessarily an argument against it. - I guess we could?

**Comment by ghost** [comment]
Unless APC power system gets reworked or you hack in an infinite APC for the whole z level, I don't see how building settlements with functioning machines will be feasible.

Perhaps make every machine function like emitters do, draining power directly from cable knots underneath them if there is no APC detected.

**Comment by ghost** [comment]
I tried to work on an Arctic Station very long ago. I scrapped everything (either way it was average code for the most part), but I still remember some important design points : 
- Realistically, the temperature can go as far down as -80°C to -90°C, which is already very hardcore (basically guaranteed to kill you unless you have all protection flags and internals)
- Average is around -50°C in Antarctica, which is basically a big frozen continent
- Ideally, outside temperature should be at the average most of the time, and randomly switch to Blizzard state which lowers temperature dramatically
- Average pressure is 68 kPa. This isn't actually enough to do pressure damage, but it will likely be far too thin to breathe. Do be careful as you might have to manually spawn snow tiles with more air
- Z-level transition in snow is very easily done by ripping off transit zones. You could have everything fully open on all sides, or intentionally funnel all directions so the transitions aren't too obvious (a glacier tile that can't be broken, or an arctic ocean tile type should both do it)
- Underground mining is something I had considered too. Add a way to build mining elevators that move you to the special mine Z-level. Make sure the Z-level is blocked on all sides (maybe with some "bedrock" type of rock)
- Snowmobiles are important. Cargo could most likely annex a snowmobile hangar for that purpose
- The Base Turf system was added explicitly so that things like the Singularity or blowing up walls can spawn any turf under them, instead of space
- Emergency shuttle can just go ahead and land on a marked landing pad and go into space. It just werks, reduces work needed for Centcomm and gamemodes massively
- Cutting down trees isn't hard at all, but it poses gameplay balance problems (how much per tree, should trees regrow naturally, be replanted, should the axe be the fireaxe, wouldn't there be problems with making fireaxes, an insanely robust weapon, publicly available)

Hypothermia/Hotspots/Protection is the biggest problem, since it needs intensive coding to be up to gear with a station that fully simulates heat difference : 
- Human base resistance to cold and heat should be MUCH, MUCH lower and increase significantly when you put on a jumpsuit so that some jackass doesn't run around outside naked, and then  proportionally depending on what body parts are covered how well. This however needs a full rework of protection flags AND clothing
- Hotspots right now are pretty shit and are only used to start plasma fires. This means fire will need to be fixed to not be fucking shit and not produce 5000°C every time you try to light a wood fire, and work outside (which is even harder given that outside tiles need to have bottomless air capacity to not be switched over). That also means an extensive ZAS rework

So yeah, it would need a lot of work; You could also go through it step by step and assume that you'll either have very cool systems with nothing to test it on, or a pretty hardcore Arctic Station where you die pretty quickly when outside unless you have a hardsuit

**Comment by FudgePucker** [comment]
Fire would have to be reworked and switched to an entirely new system for this gamemode.

The fire simulation and atmos simulation is based on you being stuck in a cramped space station with a small atmospherics, not an entire planet.

**Comment by ghost** [comment]
https://www.dropbox.com/s/g6p250z4bg6dhoz/Snow%20%281%29.dmi?dl=0

Added the crossroads, T Junction, full, side, small corner, and circle snowroads as were requested

**Comment by ghost** [comment]
Thanks @Skullyton !
@FudgePucker It's a cold planet, which means the fire behaviour does not need to change.
@dylanstrategie Thanks for your invaluable help and input. Any chance of you joining the project?

**Comment by ghost** [comment]
I have a huge backlog that I need to go through, so I can't really assist much

**Comment by ghost** [comment]
'Course you can! Never say never.

**Comment by PJB3005** [comment]
I'm so fucking tempted to drop clockcult and code this.

**Comment by ghost** [comment]
Drop everything. Snooow.

**Comment by ghost** [comment]
@Nienhaus ah, hello. How did you want your snow tile icons implemented, the 0-6? Did you have a system in mind, random, or were some tiles weighted to be more likely?
@PJB3005 I didn't even know you were still working on that. Don't feel a need to drop it though, I would feel bad if you joined the project at the expense of finally finishing clockcult.

**Comment by ghost** [comment]
@DrCelt When I made it I expected them to be all random.

**Comment by ghost** [comment]
Grand.

**Comment by ghost** [comment]
I've now coded the snow turf. 
- If you click the snow and stay still, you can make a snowball.
- If you use a shovel on it, you can clear a path.
- It is unsimulated, but the temperature and pressure are as desired. (-40C for now and ~100 kpa)
- looks pretty bitching
- if you walk in the snow you've a chance of making the comfy arse crunching noise.

Pictures soon.

**Comment by FudgePucker** [comment]
You know what, I just remembered something.

There shouldn't be any way to generate a singularity, and if one does happen it should end the round like the nuke going off.

The singularity might be able to be stopped or fuck off into space, because its just space and the space station to feed it, if the singularity was made on the planet, then it would adsorb more mass to increase its size than possibly going in the other direction into space, meaning it would eat the entire planet in under an hour, starting with the atmosphere.

**Comment by ghost** [comment]
@FudgePucker Probably a wise idea.

**Comment by ghost** [comment]
![snow](https://cloud.githubusercontent.com/assets/7193289/11581775/aacaaaf8-9a3d-11e5-8f93-3d5694ca26bc.png)

**Comment by ghost** [comment]
![h8ylvon 1](https://cloud.githubusercontent.com/assets/7193289/11582072/dc5e43aa-9a41-11e5-9977-1688578d164f.png)

**Comment by Zth--** [comment]
Walls are too square. 
I'm so fucking hyped. Fuck.

**Comment by ghost** [comment]
@Zth-- those are the normal wooden walls. If you want to make a better sprite go ahead :^)
But yeah actually, that's probably the best way to make this project finish faster, help out! If you're hyped, why not channel that into coding, mapping, or spriting?

**Comment by ghost** [comment]
Also: I need a damn name for the map. What'll it be called? This is probably the most crucial part.

**Comment by ghost** [comment]
Marooned? Crash Landing? Snow place like home? Snowzone 13?

**Comment by ihadtoregisterforthis** [comment]
If things get messy I propose changing the theme from a crash land to a extremely small and non-functional outpost. With this you can place equipment essential to players but not for building, give an explanation for shuttles to go down to the planet, somewhat explain the players that spawn in, and a few more.

**Comment by ghost** [comment]
@ihadtoregisterforthis So basically every other map, but this time in the snow?

**Comment by ihadtoregisterforthis** [comment]
EXTREMELY small
![artic icebreaker station](https://cloud.githubusercontent.com/assets/13305885/11589174/0f6669e4-9a65-11e5-91de-ce9d4f07e445.jpg)

There exist snowflake machines that need to be spawned in, or stuff no one would care to build. An example to both is a prison cell.

**Comment by ghost** [comment]
[Redacted] the snow planet.

**Comment by ghost** [comment]
@ihadtoregisterforthis we should be able to get everything we need in one shuttle.

**Comment by DrLeprikon** [comment]
Here is some concept art you could give out with the collectors edition!

 I gathered the engineers would miss the cooling opportunities space offered, so I put in some sea! 

You could have it consist of a liquid that has a very high heat capacity, so that thermal plates would work well in there, the way they would in space, except the temperature the gases in them would be approaching would have to be higher than 0K, but it doesn't have to be 0C, since the liquid is not water! You could even have the liquid be plasma, that'd make a good reason for a NanoTrassen ship to wind up there, perhaps with the plasma sea even causing the crash! 

 You could have an underwater cliff serve as the z-level boundary in the sea with only mechs and plasma_guard suits capable of staying close to it. Falling into the abyss that lies past the edge of the underwater cliff could take you to a new z-level, where there would be no natural light and mesons wouldn't work because of all the plasma, which could turn it into a hardcore asteroid for the miners!  
 Whenever the blizzard hits the sea could freeze over, lowering the temperature that the gas approaches if it is in the thermal panels in the sea, and allowing for people without plasma protection gear to walk over the frozen sea to an island z-level, which turns into a nice place when the blizzard is over! 

 The mountain could offer the regular asteroid expirience for the miners in the form of pre-built tunnels with minecarts, loot and monsters!

Thumbs up, comment, like and subscribe if you think these are horrible dumb ideas

![img_0704](https://cloud.githubusercontent.com/assets/10002997/11594785/5743cc1a-9ac5-11e5-950c-c25cb46bfde3.JPG)

**Comment by FudgePucker** [comment]
What engines would the engineers be able to set up?

Singularity is off the table, due to the previous point I made of the singularity going to stage ten and eating the entire planet before you could even say 'well shit, guess making a black hole on a planet and not in the middle of space was a bad idea.'

AME is boring and I refuse to set it up.

PACMANs would be interesting due to the whole mining prospect and the cold outside would work as a ghetto cold room to prevent them from melting down .

Atmospheric engines (TEG, SME, and R-UST if you are an ABSOLUTE MAD MAN) would need some consideration with how atmospherics would work an a planet, plasma and CO2 output shouldn't fuck over the immideate area, they would go out into the entire planets atmosphere, not exactly good for the planet, but who gives a shit, its in the middle of nowhere and an emergency. A fun bonus objective for engineers and MoMMIs should be making an engine big enough that it would turn the planet into a bright and sunny (and horribly toxic) beach resort from polution and global warming within a year. How bad of a green house gas would plasma be? It produces more CO2 than the laws of physics allows (1 mol of O2+1mol of plasma =>2 Mol's of CO2).

**Comment by DrLeprikon** [comment]
Doesn't water electrolysis produce two mols of hydrogen and a mol of oxygen per two mols of water? It's not the number of molecules that's constant in an isolated system, it's the mass and the energy

**Comment by ghost** [comment]
Boris. The implication of a fisherman in a sea of plasma amuses me. Please tell your friend to draw a fisherman riding the plasma waves.

**Comment by ghost** [comment]
@FudgePucker the shuttle crashes with a PACMAN on board though you could order the parts for a RUST with cargo.

**Comment by FudgePucker** [comment]
Oh and a shard as well right?

Will there be a 'your ship was a piece of fucking shit and we need help' discount on the shard? It is one of the most expensive things to buy because you don't want cargo to have one and blow themselves up and engineering starts with one.

**Comment by ghost** [comment]
All this talk of cargo being used for buying things, but what about stuff being located in the ruskie derelict outpost?

**Comment by lwjoestar** [comment]
Why not just call it Crash Landing? Simple, gets the point across, nobody will be confused about what they're in for when the vote screen comes up.

**Comment by ghost** [comment]
Seems like Crash Landing is the map name, then.

**Comment by ghost** [comment]
@Skullyton that's an idea we can consider, I rather like it. Ruskie Derelict is tried and tested.

**Comment by FudgePucker** [comment]
You know what, the SME would be perfect to set up depending on how the atmospherics work.
You got a very large SME chamber with it being mostly N2 and some O2, the volume of the entire planets atmosphere means that you wouldn't risk it over heating.

Just put down some catwalks around it and add an emitter and some rad collectors and baby we power now.

Gotta produce a shitton of plasma to speed up global warming to help heat up the planet.

**Comment by icantthinkofanameritenow** [comment]
Pardon to rain on everyone's parade, but you're all forgetting something very important:

The presence of antagonists on this map will make it nigh-unplayable because any damage they do to the "station" will be all but impossible to repair, especially if they go loud early. Either this map would be restricted to extended rounds and nothing else, or you'd have to make the whole thing perfectly self-sufficient from the very beginning.

**Comment by FudgePucker** [comment]
Yes but they have less means of doing damage due to the lack of toxins and atmospherics.

You think that I can't reliably make box, meta, deff, and with more effort, taxi (Little effort if I used bombs due to how small it is) nigh-unplayable?

Oops, I just attached that really fancy plasma cooling loop with scrubbers in the plasma tank to the station, and spaced the socket wrenches (hell that is even overkill, last three times I did it they didn't even notice the source of the plasma flood).

**Comment by icantthinkofanameritenow** [comment]
Yes, but at least in those cases there's the potential to rebuild because you don't have to recreate entire systems from scratch. In this case, there might not even be the materials needed to do that. You have a point about toxins, but let's all remember that Nuke Ops begin with bombs made for them and a Wizard's fireball is plenty destructive as well. Even something as petty as throwing all the station's metal into the dungeon level when nobody else is ready to explore it would bring any form of expansion screeching to a halt. 

Additionally, consider that at the start of the game Medbay will have only first-aid kits available to them at the very most, so anyone who dies early is bound to stay dead. All it would take is one guy with a murderboner and a halfway decent weapon to depopulate the station at high speed to the point the round may as well end. (At that matter, how WILL the rounds end since there's no shuttle to call?)

And now that I think of it, consider how it would  interact with antag objectives. Given the dearth of items available at first, it's possible that someone would be given an objective to steal an item that doesn't even exist. Overall, this map is a nice idea in theory but in practice it would go over like a lead balloon. SS13 is not Dwarf Fortress, nor does it need to be like DF. At the very least, make it feel like it's actually a part of SS13 instead of a spinoff that borrows a few elements from it.

**Comment by ghost** [comment]
Also btw if you guys need more spriters I will help.

**Comment by NigglyWiggly** [comment]
I can sprite some stuff too, if you need me

**Comment by ghost** [comment]
NigglyWiggly! I still don't know you but I still love your name.

**Comment by NigglyWiggly** [comment]
> I still don't know you

B-but
![ss 2015-12-04 at 10 23 12](https://cloud.githubusercontent.com/assets/9064611/11605755/ac78f9d4-9ad5-11e5-8388-a5eff9015d58.jpg)

**Comment by ghost** [comment]
Okay sorry, I mean actually talk to you.

**Comment by icantthinkofanameritenow** [comment]
Soo...nobody's taking my points into account at all? It's something I'd really suggest you think about before spriting, game balance is kind of a big deal.

**Comment by ghost** [comment]
@icantthinkofanameritenow The intention would be to have all traitor items accessible from the very start. As well as that, any key items that can not be made or built will be accessible at the start as well. If people choose to target the limited resources and destroy them, so be it, I would argue that that is a flaw with all maps rather than solely this one. In terms of how limited resources are, there will be many trees scattered around, so there will be no shortage of building materials. R&D is fully set up at the start, and hence should be more than able to produce a lot of resources themselves, and the intention is to have low value minerals accessible to mine fairly easily.

**Comment by icantthinkofanameritenow** [comment]
I suppose that's a bit more sensible. I was under the impression this would be a bare-bones map as its precursor was when we tried doing it. I'd also argue that medbay would need a cloning pod already set up as well to prevent irreversible loss of life early in the round, as well as replacing Security's stun batons with police batons to xompensate for the initial lack of power.

**Comment by FudgePucker** [comment]
@icantthinkofanameritenow 
Hell if it is on a crashed emergency shuttle then they get defib and sleepers, more than enough with proper meds to save a life.

At least force them to build a cloner, it isn't that hard.

**Comment by FudgePucker** [comment]
Alright, did some thinking.

What kind of equipment will the miners start with? They need a few ore boxes so that they can get a shitton of iron and glass. There needs to be some way to turn ore into sheets, having all three or four smelting equipment and the conveyors might be a little too big. How many mining rigsuits are there going to be?

What are engineers going to start with? Are they going to have to fight off the assistants for an easily accessible electronics crate to get yellow gloves to put down powerlines?

An idea to help solve some of the bulk of having suit storage machines would be to start off engineers, atmos techs, CE, parameds, and the shaft miners off with their exosuits, because they are going to need to spend most of their time round start anyways out in the elements.

**Comment by ghost** [comment]
@icantthinkofanameritenow I would love a bare bones map as the ideal but that inevitably relies on admin intervention, so we must be realistic and provide people with the tools to help themselves. I'm not sure about cryo to start with as it's reasonably simple to set up, and actually hold on, what's the coldness required for cryoxodone to work?

**Comment by FudgePucker** [comment]
@DrCelt 

> What's the coldness required for cyroxodone to work

Oh.
Oh god.

HYPERZINE 10u PILLS
10000000000u OF CYROXODONE/CONEXDONE IN BLOOD STREAM

LETS FUCKING CHARGE THOSE HILLS NAKED.

**Comment by icantthinkofanameritenow** [comment]
What he means is that the minimum temp is 170K or so...which may in fact be how cold it is "outside". So cryo may not even be needed if you can just dump someone outside after feeding them a few cryo/clonex pills.

**Comment by ghost** [comment]
How's about limited stuff in the crashed ship Z-level (The ship itself in one corner, medical shuttle with medical basics in another corner, mineshaft, etc.) and have some of the higher tech stuff in a derelict base on another Z Level (Hardsuits, cryo, cloning, R&D servers, etc.)

On top of that, extreme cold in atmospheric conditions freezing up hardsuits when?

**Comment by FudgePucker** [comment]
@Skullyton 
That would make some projects an absolute shit to work on like tcom projects and the like.

**Comment by ihadtoregisterforthis** [comment]
_If we only had ore redemption machines..._

**Comment by ghost** [comment]
Cryo would need temperatures below -100°C to work. If you restrict the coldest temperatures to about -90°C, it should work fine

**Comment by Zth--** [comment]
Guys don't forget that this is a survival map, if people don't cooperate they should die at 20-30 minutes because of random events/coldness/etc. If not, people will get bored really soon.
About systems, we have botany and ghetto chemistry for medbay. Advanced stuff should be available rarely or never.

**Comment by butthatswrongyoufuckingspiral** [comment]
Going to be signing up as a mapper once we get a clear picture of what we want. I'm totally hyped for the abandoned colony concept. What I'm wondering is, is there going to be an end-game, for example, getting to the point where we construct some form of escape shuttle to leave the planet? I don't think it would be very exciting to just hail CentComm and get rescued so, maybe make it canon that the planet's atmosphere or whatever disrupts communications to and from the planet.

**Comment by NigglyWiggly** [comment]
@Zth-- 

> if people don't cooperate they should die at 20-30 minutes 

This, as comfy as it would be to have your own little secluded snowy cabin and stuff, the point of this is that it's not exactly a comfy situation, I assume. You should be working together to stay alive, maybe scavenge parts of your ship to build radio equipment to hail CentComm through the macguffin field that messes with your radio?

EDIT:
![ss 2015-12-05 at 08 16 34](https://cloud.githubusercontent.com/assets/9064611/11611163/22f488fa-9b8d-11e5-86bd-306771698583.png)

**Comment by DrLeprikon** [comment]
@butthatswrongyoufuckingspiral 
That is exactly what a sea of plasma would do! When a sufficiently powerful transmitter, the command console for example, attempts to contact a distant NanoTrassen relay node, it releases an electro-magnetic wave, which is comparable to a burst of gamma radiation, it excites the plasma sea the way a radiation storm would, which makes the substance evaporate and, since it is a rather dense gas, form an opaque cloud, that would slowly enroach on the crashed ship, seeping in through the tiniest of cracks and behaving as though any material were a semipermeable membrane, eventually suffocating the stranded men!

 Is that any good? Can I be the Nestor of spess?

**Comment by icantthinkofanameritenow** [comment]
@Zth:

Shouldn't there be a way to end the round quickly just in case the macguffin isn't found before shit hits the fan (e.g. a traitor enters murderboner mode right off the bat before anything gets set up and destroys the bodies so they can't be cloned)? Without adminbus intervening, I mean.

**Comment by ihadtoregisterforthis** [comment]
Given that this is a survival mission, that it will be rare, that many players will flee for the entire round, and because all the balance and rethinking problems, this map might as well not have antags.

**Comment by icantthinkofanameritenow** [comment]
@ihadtoregisterforthis

I brought that up earlier. While that's the best solution, it would also mean that the map would have to be locked of regular rotations as a result. You'd either need to bus it to make sure it only triggers on Extended (and turn random events off, the prototype version failed since there were no maintainence tunnels to escape a radstorm from) or host it on its own server. Neither of those is what I would call a long-term solution to the issue.

**Comment by ghost** [comment]
It is a dilemma, yes. The way I see it is there are two main options
- Full extended, no way to call for help - this is not vg's ss13 and certainly could not be placed on the map rotation
- Call for shuttle possible, antags enabled, map rotation on. - This detracts from the "stranded" element - could take longer (?)

Honestly I'm in favour of the map rotation trouncing all else in regards to this, but I'm interested in your input and will ask the thread for theirs in a time that isn't dedpop.

**Comment by ghost** [comment]
Okay so what I was thinking was you guys could add a new type of generator that can burn coal or charcoal called the Integrated Coal/charcoal burning generator, ICB Generator. Make it that if you cut down a tree along with making a fire-pit and burning it for at least 10 seconds and putting it out you get charcoal for the ICB. Along with there should be a Russian or some other misc group that has a small outpost station with just enough supplies for maybe 4-5 people? and it has an underground level, can use the mining elevator sprites, and it leads down to a partly mined out cave system. Oh and maybe have like 4-5 dead bodies randomly spawn around the map with supplies.

**Comment by NigglyWiggly** [comment]
> Not wanting antags on this map

Guys guys 
Did no one remember The Thing?
You'd be stranded with this horrible shapeshifting creature in the middle of nowhere with no way to call for help, and you can't just run out into the snow because the cold will kill you, I don't really know how our changeling mode would work with this, what with the slow absorbing and all, but it could be fun, you wouldn't be sure who you could trust but because of the hazards you couldn't just go out alone

**Comment by icantthinkofanameritenow** [comment]
@NigglyWiggly

Beyond the obvious fact that nobody in The Thing was looking for their valids, they also had an advantage by way of that one blood test that could prove whether or not somebody was a Thing. We don't have an equivalent of that on our server, so it would devolve very quickly into "kill everyone else, just to be safe". And that's assuming that Security was present and relatively competent. If they aren't around, I can imagine things quickly turning into an every man for himself situation. Plus, with no shuttle present how would the round end?

**Comment by NigglyWiggly** [comment]
@icantthinkofanameritenow 
We could make a little machine to test blood, that way it can be on this map only and not a common thing, and they didn't need Security in The Thing, they just all ganged up on Blair and tossed him in a shed. Besides, I'd assume most of the crew would be armed, not just security, right?

**Comment by icantthinkofanameritenow** [comment]
@NigglyWiggly

The blood testing machine being a unique part of the map could work, as long as it was made in a way that its deconstruction or destruction was somehow impossible. Otherwise smarter lings might try to get rid of it or sabotage it. Assuming it was a traitor+ling round, an e-mag could rig it so the result was set to generate a false negative if the users blood is tested- or it could be made to give positive results for everyone even if they aren't changelings. 

As for making everyone armed...do you really trust the greytide to have the self-control needed to not shoot anyone they don't like?

**Comment by NigglyWiggly** [comment]
@icantthinkofanameritenow 
Deconstructing the machine would be bad, yeah, but sabotaging it is always nice, the emag stuff could be good for that.
The issue of arming the crew is a kind of debatable one, on one hand, yeah, I'd hate for every greytider to have the means to kill anyone they want with ease, but at the same time, there are supposed to be hostile mobs outside, right?

**Comment by icantthinkofanameritenow** [comment]
@NigglyWiggly

I'm not arguing with that. I just don't trust a lot of people to resist the urge not to turn their weapons on each other. Besides, they can always fall back on spears or the like at the very least.

**Comment by ihadtoregisterforthis** [comment]
The bunsen burner awaits

**Comment by Blithering** [comment]
![transparent orange chainsaw](https://cloud.githubusercontent.com/assets/15067109/11703062/db305ee6-9ed3-11e5-9d7d-a93278e42b70.png)
I would be up for some spriting work. I did have a snow/Hoth stormtrooper in the works to go with the big mess of stormtrooper sprites I was making, but I'd probably be best waiting until the lot that are already have been finished are put on the github by some kind soul before throwing more on the pile, or it'll never stop growing. Let me know what still needs working on and I'll give them a shot. Since you're on an ice planet you're going to be needing a TRANSPARENT ORANGE CHAINSAW. Slidey slider has been slid, transparent orange chainsaw now lives up to the hype.

**Comment by ghost** [comment]
@Blithering while I can get around to coding pretty much any sprites into the game (I'll code your sprites if you sprite my code baby~), the stuff I believe is necessary for now is listed in the OP and if there's a name on there beside it someone's taken that job already. - but I'd fucken love to have hoth sprites in the game for this honestly

**Comment by Blithering** [comment]
Guess I'll take ship debris then. Are we using the standard white shuttle tileset as a base?

**Comment by NigglyWiggly** [comment]
@Blithering I'd go with a darker tone, just for some contrast with the snow

**Comment by FudgePucker** [comment]
Got it, give the escape shuttle scorch marks from reentry into the atmosphere.
The escape shuttle wasn't made to enter an atmosphere and is now darker with black streak marks on the walls.

IDEA
GENIUS

**Comment by ghost** [comment]
@fudgepucker

> reentry

But if it's never been on the planet before how can it re-enter?

**Comment by ihadtoregisterforthis** [comment]
@Blithering But is it transparent?

**Comment by DrLeprikon** [comment]
@DrCelt 
It's not reentering the planet, it's reentering a medium that is an atmosphere, doofus! I still remember you trying to convince me someone aboard the station can't hear someone pressing their helmet against the window, which is also wrong! Can I be the lead physi-realisics of this project?

**Comment by FudgePucker** [comment]
The smart ass just got smart ass'd
Thanks @DrLeprikon

**Comment by Blithering** [comment]
@ihadtoregisterforthis Unfortunately not. Haven't worked out how to get transparency on dreammaker yet. It will become transparent as soon as I figure that part out, then we can all be the lego man.

**Comment by ihadtoregisterforthis** [comment]
Select a color and slide the slidey slider on the side of the palette

**Comment by ghost** [comment]
https://www.dropbox.com/s/6xmuquvmobmvh0g/snow2.webm?dl=0
making webms is hard. pretty self explanatory what this is I explained more in the pull request but figured I may as well cross-post it here

**Comment by Wizardcrying** [comment]
The temp change seems WAY too fast. If you strip your coat and get near the fire, putting your coat back on afterwards, you should be able to go quite some time without having to be near a campfire again. This would be good for any sort of antagonists that get bussed in somehow to give them some room to operate so long as the whole area isn't developed.
"The Thing: WE BUILD Spess Edition" was done before, and I'd like to see it done again with these changes.

**Comment by ghost** [comment]
@Wizardcrying the intention is to rehaul the way temperature change is done, yep.

**Comment by Blithering** [comment]
So I'm doing a damaged version of the darker shuttle tileset for mixing and matching to make the crash, and I have a couple of ideas for set pieces and random debris to have scattered around in it's wake. Does anyone have anything they'd like/expect to see in a crash?

**Comment by ghost** [comment]
Most likely fell trees, a large impact trail and a bunch of scattered debris and shipments. All debris should be removable to a degree (just welding them into iron seems fine)

**Comment by Blithering** [comment]
Did a closet portal sprite btw, seeing as the code for it was finished. hopefully it doesn't look too much like a butthole. http://www.filedropper.com/closetportal

**Comment by Blithering** [comment]
I did promise Celt to upload something asap, and the debris is taking longer than anticipated, so here's something quick and direly needed. They were roasting doughnuts on the fire out of desperation during the comfy time in the caps office the other day, the poor souls.
http://s000.tinyupload.com/index.php?file_id=17423275684443618339
Marshmallows! With or without stick. Edit: nearly forgot a sprite for hot chocolate with marshmallows.

**Comment by unid15** [comment]
So who's making the map

**Comment by ghost** [comment]
hahaha ahaha nobody.

**Comment by ihadtoregisterforthis** [comment]
![snap](https://cloud.githubusercontent.com/assets/13305885/11910504/dd2b87fc-a5d6-11e5-9b76-10a8536a1d7b.png)

**Comment by ghost** [comment]
The assets need to be finished first before a map is started

**Comment by Blithering** [comment]
Hope everyone had a good xmas, here's a couple of sprites. 
Updated stormtrooper file with snowtrooper, also blaster now has a light to show the charge left in it.
http://s000.tinyupload.com/?file_id=06407493695475829304

Tiny fluffy mothman/mothmonkey for possible use as a native NPC, or just general fluttering.
 http://s000.tinyupload.com/?file_id=01054187951625794753

**Comment by Blithering** [comment]
Wall redo for Celt. hope this is the right one, turns out i had two vgstation files.
http://s000.tinyupload.com/index.php?file_id=43546811461047438273

**Comment by NigglyWiggly** [comment]
Woah shit I just noticed, Blithering was that a TRANSPARENT ORANGE CHAINSAW?

You are the best.

**Comment by Blithering** [comment]
Can't go ice mining without one of them Niggly.

Thanks for reminding me to upload it here
http://s000.tinyupload.com/index.php?file_id=69684567333092856189

**Comment by ihadtoregisterforthis** [comment]
It's 2016, people

**Comment by ghost** [comment]
> Snow minerals

I didn't realize nobody had taken it up, hang on while I get that sorted.

**Comment by AlphaBuffalo** [comment]
If I had my computer I would help make a map, my idea for the main z-level would be a crashed ship full of stuff next to an abandoned base, there could be a building for each department, and plenty of free space to build, but there wouldn't be power at first (so engineering has to be competent) and all the buildings would be intact, but obviously in a state of neglect i. e. Cob webs, busted lights, busted computers, pulled floor tiles, dusty, and so on. And in the case engineering is incompetent there could be super pacman generators that can be powered once mining gets uranium

**Comment by AlphaBuffalo** [comment]
okay how would cargo work when the crew crash lands on a planet and centcom has no idea where the crew is? How about the map becomes reclaim map and crew lands gently on the planet and centcom wants us to rebuild the base and do plasma shit

**Comment by ghost** [comment]
I preferred the crash landing idea though

**Comment by AlphaBuffalo** [comment]
BUT HOW CAN CENTCOM SEND A SUPPLY SHUTTLE WHEN THEY HAVE NO IDEA WHERE THE CREW IS? Unless crew crash lands because the captain is retarded in which case that could work

**Comment by AlphaBuffalo** [comment]
Like its a reclaim mission, captain pilots the shuttle, he crashes next to target but dies while everyone else is okay

**Comment by FudgePucker** [comment]
Maybe an emergency one way cargo telepad. It can only receive one way, only one crate at a time, and on a cool down. Requires cargo to order things with more consideration.

**Comment by AlphaBuffalo** [comment]
whats stopping centcom from teleporting a tracking beacon?

**Comment by FudgePucker** [comment]
Tracking beacons have a specific range or some shit

**Comment by ihadtoregisterforthis** [comment]
It's supposed to be about WE BUILD, not WE PURCHASE.
Materials and equipment are OSP.

**Comment by ghost** [comment]
OSP?

**Comment by ihadtoregisterforthis** [comment]
On-site procurement

**Comment by AlphaBuffalo** [comment]
So cargo will be useless?

**Comment by FudgePucker** [comment]
Maybe require there to be a functioning telecommunication set up with comms computer to have the pad function, requiring set up before orders can be made

**Comment by AlphaBuffalo** [comment]
Someone needs to code that in

**Comment by ghost** [comment]
I can code an interstational relay and have it a toggle on a map

**Comment by AlphaBuffalo** [comment]
So how would the round end? Crew build a special relay and call shuttle?

**Comment by ghost** [comment]
1. Interstational relay can be built, using it will link you with Central Command
2. If you do not build it within half an hour, Central command finds you anyway.
3. Crew can call shuttle of their own accord, if they really want to leave I guess but there's no need if they're researching plasma

That work fine?

**Comment by AlphaBuffalo** [comment]
Make the relay easy to build, like just powering a machine

**Comment by AlphaBuffalo** [comment]
Maybe make it so it works after it charges up

**Comment by ghost** [comment]
I'll code the relay later, but that's the code for not calling shuttles and suchlike done

**Comment by AlphaBuffalo** [comment]
Going against popular opinion here, but would the people who play this server really want to repair or build their work place just to do their job? It kinda seems like adding extra steps to playing the game

**Comment by ihadtoregisterforthis** [comment]
How about having "difficulties", where harder means scattered random buildings and the easiest start with some base?

**Comment by AlphaBuffalo** [comment]
The map I'm making is an abandoned base, but most of the stuff needed is in the shuttle, the base will have rooms where people can use what's there at to do limited work, or they can put some effort and do everything they want. Like the medbay area wont have cryo but just sleepers and some med kits, the only functional place will be the AI core where you just need to move the AI too

**Comment by Blithering** [comment]
Sprites for crashed shuttle walls, and random detritus from the crash http://s000.tinyupload.com/index.php?file_id=41353808696419994449

**Comment by Blithering** [comment]
Also I had a thought about the way a round could work. Instead of starting from no power etc, the shuttle could start with functional but basic medbay, but with a finite power supply. It could deal with the very early issues from falling trees, space bears and wendigos, but as you get set up, the onus is on the engineering team to move the machines to a more permanent location.

**Comment by MusgardTiger** [comment]
Hello, Quartermaster and cargo should be in charge of materials. Like processing raw materials should be cargos work. So they will have a job and a very important one. Let quartermaster assign materials to departments (finally quartermaster a proper head role)

Also I'll be back with sprites of friggen animals.

**Comment by ghost** [comment]
@Blithering I actually like that idea.

**Comment by MusgardTiger** [comment]
https://www.dropbox.com/sh/61t3qvimb5cdavu/AABq5Sm9-J2flQ5JFEl00pW7a?dl=0

Shrew, a critter. Should drop a piece of meat slab when butchered. Also I don't know if I should make woodland critters more elaborate or keep the sprites really simple. Pls feedback.

Edit: https://www.dropbox.com/sh/uwzsrlmknwty8oz/AAApuz1hxB7gX6mMkRhU8kp2a?dl=0 (Smaller sprite, stole mouse front sprite for size reference)

![shrew-small](https://cloud.githubusercontent.com/assets/10194313/13508689/b759bdc8-e190-11e5-8125-89c4467e84ac.png)

Update2: https://www.dropbox.com/sh/if2042y4xo3r2kl/AACDzEnvb74q8_BtQPyApTBma?dl=0 (Fox)
Edit2: Edited Fox left-right sprites.

![fox](https://cloud.githubusercontent.com/assets/10194313/13510932/ba4ecbd0-e19b-11e5-91c1-4e42ab739d06.png)

**Comment by Blithering** [comment]
Simpler the better in my opinion. It allows the sprites to be smaller, and keeps with the current mobs aesthetic.

**Comment by ghost** [comment]
What's some good ideas for tundra mobs then for hunting?

> Deer
> Penguins
> Rabbits

**Comment by Blithering** [comment]
Space rabbits! With a vicious streak a mile wide!

**Comment by ghost** [comment]
On it like a bonnet.

Go with the mouse colourings? Already got a white-with-red-eyes sorted almost, just gotta do the front, dead, and maybe sitting sprites if we want to go cat fancy

**Comment by ihadtoregisterforthis** [comment]
Foxes, vicious space beavers, wolves, and pelt hats from all of them.
Add some alien creatures so it isn't just alaska-station with penguins.

**Comment by ghost** [comment]
![Rabbit](http://i.imgur.com/eRHvVWD.png)

3 shades, white with red eyes, brown with darker brown-red eyes, and grey with brown eyes

https://www.dropbox.com/s/yd7n112jwl9qgyd/rabbit.dmi?dl=0

**Comment by MusgardTiger** [comment]
he is standing on his tippy toes aaaaaaaaaaaaaaaaaaaaaaah 10/10 sprite.

**Comment by Blithering** [comment]
Possible sprite for wendigos, maybe tie in to cannibalism. http://s000.tinyupload.com/index.php?file_id=09004035935187717662

**Comment by FudgePucker** [comment]
We can "borrow" some goon sprites for mobs. They got ice moon mobs

**Comment by ghost** [comment]
What about a condition where, if you start eating human flesh while on map as a human, you start to go wendigo?

Because holy shit I want to sprite this so bad, or maybe just have it as an alpha wendigo? ![Wendigo](http://vignette2.wikia.nocookie.net/powerlisting/images/3/32/Wendigo1.jpg/revision/latest?cb=20120213141914.png)

**Comment by Blithering** [comment]
Maybe have that as the wendigo equivalent of horror form? I was trying to make it look like a walking frozen corpse, and have those playing as wendigos rely on full body coverings to avoid detection, but giant deerbonimation would be fun.

**Comment by NigglyWiggly** [comment]
Wendigos aren't really a stealthy thing, are they?
I always assumed the only stealth a wendigo did was hiding while stalking its prey

**Comment by Blithering** [comment]
Reading up on wendigos, one bit could work. 'Whenever a Wendigo ate another person, it would grow in proportion to the meal it had just eaten, so that it could never be full.' Maybe have several stages of wendigo growth as they eat more people, starting as a human and ending up as a spider queen sized monstrosity.
![wendigo edit](https://cloud.githubusercontent.com/assets/15067109/13560665/ca790068-e419-11e5-84bf-ec378aec367f.png)
Edit: Result of a bit of messing around in alienqueen.dmi. Colours are all wrong but you get the idea.

**Comment by NigglyWiggly** [comment]
That looks great.

**Comment by Blithering** [comment]
Regarding the problem with the logic errors for cargo's ordering, the issue could be sidestepped by having the ordering computer be an abandoned syndicate/ruskie supply terminal. 
Automated supply drops can slip under the radar, but if the crew attempt to signal a rescue craft the organisation sends a shuttle full of russian or syndicate mobs, being a little miffed at being duped out of material. The shuttle can be a risk/reward system, as if you are a bad enough dude to clear it out, you get to loot the bodies for questionably useful suits, and definitely useful guns.

**Comment by ghost** [comment]
What about some sort of generic clothing crafting system, using the already present skins you can butcher off mobs?

Could create a set of clothes that use a generic leather sprite, that just inherits the name of the source, I.E Corgiskin breastplate, Xenoskin shoes, Bearskin hat

Then, of course, you could have special alternate sprites for this armour if needs be.

This'll give more of a reason to properly process your kills (Do it clean, you get the leather and more meat, do it rough, you get bare minimum meat and no leather) and to actually go out hunting

**Comment by Blithering** [comment]
Hunting parties would be great for turning what is usually a nuisance into an important part of survival. Greytide with blunderbusses and spears grouping up to take on bears to feed the station.

**Comment by NigglyWiggly** [comment]
@Skullyton 

> Then, of course, you could have special alternate sprites for this armour if needs be.

I might be willing to do this if I get a chance, sounds like it could be neat.

> Wearing a corgi or xeno head as a hat

**Comment by ghost** [comment]
Do it @NigglyWiggly

**Comment by ghost** [comment]
Heck, could have certain odd effects for different leather (Xeno leather clothes are acidproof, bear leather clothes give you better heat and cold protection, etc.)

Guess that'd make hunting the larger enemies more worth it, as I can't think of any special effects that would make sense for, say, monkey or corgi

**Comment by Blithering** [comment]
Did a set of icons for joining snow barricades together, for making igloos with. Should probably lower the cost of making them if they're going to be used to make structures, in my experience collecting enough even for a simple wall was a pain. link to download for snow walls: http://s000.tinyupload.com/index.php?file_id=37146154985940061915

**Comment by Blithering** [comment]
Possible wendigo sprite: http://s000.tinyupload.com/?file_id=00391869441201998231

**Comment by Brolaire** [comment]
what happened to this

**Comment by ghost** [comment]
It died, all the resources are here there and everywhere

**Comment by Blithering** [comment]
I'm still up for more sprite work for this, if anyone else is doing stuff. Haven't seen Celt online in a while, and it was his project.

**Comment by Brolaire** [comment]
@DrCelt

**Comment by icantthinkofanameritenow** [comment]
@Brolaire 
It died. 

Does this mean we can finally get rid of that stupid hypothermia shit?

**Comment by ghost** [comment]
@Brolaire Soon. I'll be back.

As to icant, hypothermia stands on its own two feet- previously body temperature was 20°c. If nothing else hypothermia has created a more realistic temperature management system in the game, and sure there were hiccups and I accept more than anyone else it wasn't perfect, I think you've been overreacting to the nth degree since the very start as to its importance or problems.

**Comment by icantthinkofanameritenow** [comment]
@DrCelt

Perhaps. I simply believe it was unwise to fix something that until that point was never considered an issue to begin with. Even more so since the feature it was made for the sole purpose of supplementing has gone for months without even a bare-bones prototype being made. Ideas and brainstorming are all well and good up until a point, but without any results being shown after nearly half a year I can't help but feel like it's doing more harm than good. Better to get to work actually proving that it can work first and then add in all the features that you want to shoehorn in. And of course, do us the small favor of making it all feel like it actually belongs in the game and isn't just tacked on there for its own sake. 

Plus, the thermal homeostasis regulator no longer has a purpose now that cryo itself thermoregulates; it's little more than a waste of space (which last I checked still couldn't be deconstructed). Granted, it might be tolerable if the sprite was half its current size so there was more room in the already cramped cryo chamber, but that alone won't do much to justify it. You might as well replace it with a giant alarm clock since that's its only real function now. 

But to change the subject, why is everyone trying to push cannibalism as some kind of innovative feature? It's been in the game for ages, courtesy of the chef's gibber. And at that matter, there's no motivation to hunt or even eat if hardcore mode isn't forced every time this map is selected- why worry about survival when starving does nothing but slow you down? I'm sure there's a way to force it on without constant admin intervention to do so, but I haven't a clue what that may be. 

But in all seriousness, I think it would be better to make a working version of the map as a proof that the basic premise works, even if it means cutting out planned features for it. Brainstorming is fine up to a point but unless it gets put into practice it just ends up wasting everyone's time.

**Comment by ghost** [comment]
So, we got Wendigos, snow crabs, polar bears, penguins, rabbits, and snow corgis once I've got the coloring right

Anyone got any Deer?

**Comment by ghost** [comment]
https://www.dropbox.com/sh/cakk6sqntoqpfc3/AABcXj5nPBSmMAa1yfbjhwJ7a/DMIs/Deer.dmi?dl=0
Deer sprite right here.

**Comment by ghost** [comment]
Sweet. That's bucks out of the way, but what about doe?

> Flesh deer

Jesus christ

**Comment by ghost** [comment]
I only have bucks in my life ;). I can try a doe when I get home.

**Comment by ghost** [comment]
Aighty, thanks Nienhaus

**Comment by ghost** [comment]
@Blithering Any chance I can get those chainsaw sprites?

**Comment by Blithering** [comment]
@Skullyton  http://s000.tinyupload.com/?file_id=64779906207415076804 They're in there.

**Comment by ghost** [comment]
Not gonna lie, I forgot.

On Monday, May 16, 2016, Blithering notifications@github.com wrote:

> @Skullyton https://github.com/Skullyton
> http://s000.tinyupload.com/?file_id=64779906207415076804 They're in
> there.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/d3athrow/vgstation13/issues/6789#issuecomment-219491814

## 

-Cortex Computer Assistance Co-Founder

**Comment by icantthinkofanameritenow** [comment]
Tell me, is anyone actually making the map at all now? It's been half a year now and all it's produced are ideas that go nowhere, a couple of sprites, and of course hypothermia. 

And all without even starting up even a preliminary sketch of what anyone expects he map to look like. I don't know about you, but perhaps the people working on this (assuming they're doing something other than making more ideas which will never be implemented) should reexamined their priorities and get the map in a functional state and then add all the gimmicks.

**Comment by ghost** [comment]
@icantthinkofanameritenow 
k

**Comment by icantthinkofanameritenow** [comment]
@DrCelt

It's just basics, man. Can't build a house without a foundation.

**Comment by ghost** [comment]
Houses don't get built by people sitting on the sidelines either.

**Comment by icantthinkofanameritenow** [comment]
Nor are they built solely by talking about how great all the furniture will be or how amazing the wallpaper will look but never getting started on the actual building. 

There will be plenty of time to spruce it up with critters and scenery later, it's more important to actually set up the map first. As I currently lack the time.to map it myself, I suppose I'll put in some concrete guidelines so this doesn't once again get sidetracked by people going "wouldn't it be great if..." and thus putting this even further away from  any actual implementation. 

So to start, I envision the crashed ship as being at or near the center of the map. Any items absolutely necessary for survival (e.g. a PACMAN generator for power, mining equipment and smelter for raw materials, hydroponics for food, etc.) should be relatively close to the initial crash, with other goodies like Security gear and an autolathe scattered further away but not so far as to make retrieving it genuinely dangerous. Generally, the more valuable the stuff, the farther away it's landed and the harder it is to reach safely. (Note that as in this scenario some antag objectives will be unavailable, those objectives to steal unavailable items should be disabled somehow.)

For the sake of being able to end a round if it's not possible for an admin to do it directly, the comms console should be able to send a "distress call" to CentComm, which will deploy a shuttle somewhere on the surface. This surface should also be clearly marked so nobody accidentally builds on top of it. While it shouldn't be possible to recall it, the shuttle should take longer than usual to arrive to compensate  should this somehow be run with antagonists. 

The main Z-level perimeter should be either marked by an impassible barrier (call it very dense forest), or alternately make it so crossing one edge of the Z-level loops you to the opposite edge. Planets are round, after all. 

If possible (and I know you want to do this anyway), force-enable hardcore mode anytime the.map is selected. Otherwise the "threat" of starvation is just an annoyance. 

You will want to make several jobs non-selectable for the duration the map is being played, since they will have no purpose or correspond to nonexistent departments. No Atmospherics system means atmos techs will be worthless, for example. 

See what I mean? You start with the barest essentials to make the map concept function, and then you add all the other doodads like cave systems and hostile creatures after all of that is done and not a moment before.

**Comment by ghost** [comment]
@icantthinkofanameritenow
That's actually some constructive suggestions and certainly should be kept in mind, thanks.

**Comment by icantthinkofanameritenow** [comment]
@DrCelt

You are welcome. Just do me a favor and see to it that something gets made that resembles a map at some point. I'll make a crappy sketch of the map in Microsoft Paint if that makes things easier to map out, as long as it moves things away from brainstorming and into tangible results. 

I get impatient with ideas that prove to be nothing but empty talk, and after this long I am compelled to wonder if this too will amount to nothing in the end. I'm sure nobody participating in this discussion would appreciate the idea of all their brainstorming being for nothing- I certainly don't.

**Comment by Megamaw** [comment]
so is this idea deader than my dreams for playing as a cool skeleton janitor?

**Comment by ghost** [comment]
Pretty much

**Comment by AlphaBuffalo** [comment]
Its still possible but the main set back is that celt has been gone for so long.

**Comment by ghost** [comment]
@AlphaBuffalo https://www.youtube.com/watch?v=K9q70zCJYBc

**Comment by AlphaBuffalo** [comment]
Get to work then, wasting my time watching youtube videos...

**Comment by Konater** [comment]
@DrCelt Too many snow tiles and dream daemon freezes when you start up the server, any idea why?

**Comment by NigglyWiggly** [comment]
@Konater Too cold

**Comment by ghost** [comment]
@konater I need to move the New() proc to Initialise()

**Comment by Probe1** [comment]
Old discussion.

**Comment by ghost** [comment]
Old discussion that's still relevant tho

It's got so many frikkin sprites

**Comment by Probe1** [comment]
Put the sprites in a new issue so we know they exist.
